src/components/Header/Header.jsx

```diff
@@ -0,0 +1,16 @@
+import styles from "../../css/Header.module.css";
+
+function Header({ onChangeAddModal }) {
+    return (
+        <>
+            <header className={styles.gnb}>
+                <h1 className={`${styles.gnb__title} text-title`}>점심 뭐 먹지</h1>
+                <button type="button" className={styles.gnb__button} aria-label="음식점 추가" onClick={onChangeAddModal}>
+                    <img src="../templates/add-button.png" alt="음식점 추가"></img>
+                </button>
+            </header>
+        </>
+    );
+}
```

Comment:
```suggestion
function Header({ onChangeAddModal }) {
    return (
        <header className={styles.gnb}>
            <h1 className={`${styles.gnb__title} text-title`}>점심 뭐 먹지</h1>
            <button type="button" className={styles.gnb__button} aria-label="음식점 추가" onClick={onChangeAddModal}>
                <img src="../templates/add-button.png" alt="음식점 추가"></img>
            </button>
        </header>
    );
}
```
이 컴포넌트에서는 Fragment가 없어도 될 것 같아보이는군요?

Reply:
오 정말이네요... 감사합니다👍